themes/vue/source/css/_ad.styl

```diff
@@ -30,41 +30,37 @@
     font-weight: normal
 
 // text ad (below page title)
-.bsa-cpc
-  font-size 1em
-  background-color #f8f8f8
-  a._default_
-    text-align left
-    display block
-    padding 10px 15px 12px
-    margin-bottom 20px
-    color #666
-    font-weight 400
-    line-height 18px
-    .default-image, .default-title, .default-description
-      display inline
-      vertical-align middle
-      margin-right 6px
-    .default-image
-      img
-        height 20px
-        border-radius 3px
-        vertical-align middle
-        position relative
-        top -1px
-    .default-title
+#bsa-native
+  .native-box
+    box-shadow inset 0 0 0 1px hsla(0, 0%, 0%, .075)
+    background-color: #f8f8f8;
```

Comment:
You can remove the colon in these rules btw.

Reply:
@phanan thanks for reviewing! I’ve removed the colon in these parts. Do we want to remove all the colons in this stylesheet?

Reply:
I'd say it would be nice for you to do so, as long as we don't break anything :D

Reply:
Please review one more time. Should be good! :)